readme.md

```diff
@@ -4,13 +4,13 @@ clone the repository to get a copy of the code on your device, and then code can
 
```

Comment:
this commit doesn't feel like it should be part of this PR, maybe it seems silly to give it its own PR, but I think that would be ok, to split it out, just cherry pick this commit and apply it to master, push to origin master and you can make a pr from your master to this master and also have another one from forward_declarations into master as well. (Once you have cherry picked the commit and it is in master, you can just remove this commit as part of a rebase and then do a force push)

Reply:
If you were going to leave it in the PR and only do a single one, I would at least put this commit first using rebase so that I the reviewer am not distracted by it while I am in the middle of reviewing the main code changes.